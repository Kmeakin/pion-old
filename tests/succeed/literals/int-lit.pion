//~ run = "${PION} elab ${FILE}"

{
let _ = 0;
Type
}
