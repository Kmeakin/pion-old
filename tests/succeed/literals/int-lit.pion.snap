exit status: 0
stdout:
let _ : Int = 0;
Type : Type

stderr:
